tests/sentry/feedback/usecases/test_create_feedback.py

```diff
@@ -190,7 +190,6 @@ def test_fix_for_issue_platform():
     fixed_event = fix_for_issue_platform(event)
     validate_issue_platform_event_schema(fixed_event)
     assert fixed_event["contexts"]["replay"]["replay_id"] == "3d621c61593c4ff9b43f8490a78ae18e"
-    assert fixed_event["tags"]["user.email"] == "josh.ferge@sentry.io"
```

Comment:
why'd this get removed?

Reply:
It's covered in `test_create_feedback_tags`. It'd still work if we kept it, but the new test tests things work e2e, regardless of `fix_for_issue_platform`.

Reply:
Actually I just moved the logic to create_feedback_issue. The helper was getting overloaded with tags stuff